.github/workflows/connect-dev-release-test.yml

```diff
@@ -19,6 +18,7 @@ on:
       - "submodules/trezor-common"
       - "yarn.lock"
       - ".github/workflows/connect-dev-release.yml"
+      - ".github/workflows/template-connect-popup-test-params.yml"
```

Comment:
btw is there any way how to share this list of paths between workflows? we are already duplicating it with other workflows

Reply:
I was researching that but GitHub Actions does not directly support importing or inheriting the on trigger configurations from one workflow file to another. Each workflow file is considered independently.

There are few things we could do to don't have it duplicated, we could create one workflow for connect unit test and connect e2e that will have a single `on:` trigger.